mobile/lib/modules/album/views/album_viewer_page.dart

```diff
@@ -248,32 +248,45 @@ class AlbumViewerPage extends HookConsumerWidget {
       );
     }
 
+    Future<bool> onWillPop() async {
+      final isMultiselectEnable = ref.read(assetSelectionProvider).selectedAssetsInAlbumViewer.isNotEmpty;
+      if (isMultiselectEnable) {
+        ref.watch(assetSelectionProvider.notifier).removeAll();
+        return Future.value(false);
```

Comment:
Just curious, if we return `false` here, would it be converted to a Future since it is indicated in the method signature already?

Reply:
Yeah you can do that, but you have to add `async` to the function signature.